packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -76,6 +77,8 @@ export interface BasicCrawlingContext<
      * @returns Promise that resolves to {@apilink BatchAddRequestsResult} object.
      */
     enqueueLinks(options?: SetRequired<EnqueueLinksOptions, 'urls'>): Promise<BatchAddRequestsResult>;
+
+    pushData(...args: Parameters<typeof Dataset.pushData>): ReturnType<typeof Dataset.pushData>;
```

Comment:
lets add some comments here as well, same as in the context helper

Reply:
I suppose we don't need to have this here, as it's basically the same as the inherited `CrawlingContext.pushData`.